Mobile/iOS/Templates/xxx-share_edit_privileges.md

```diff
@@ -0,0 +1,49 @@
+### Sharing privileges
+
+#### Pr: https://github.com/owncloud/ios/pull/xxx
+
+Devices:
+
+Server version:
+
+<br>
+ 
+Test Case ID | Test case     | Steps   | Expected behavior | Result | Related Comments
+|:----:|:------------- |:-------------------|:-------------|-------------:|----------:
+**Menu views**|
+1|New file sharing options| 1. Swipe to share with users/groups<br>2. Tap on "add user or group" and add one of them<br>3. Tap on handle privileges|2. A badget is shown to handle privileges<br>3. Two options are displayed: "can share" and "can edit", a "Stop sharing" option, and a "Done" button to confirm. On the top appears the name of user/group to share with
```

Comment:
I mean the options that are going to be added. In this test case i will only check that all of them are correctly displayed,